content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -401,7 +401,12 @@ Example: `service.kubernetes.io/topology-aware-hints: "auto"`
 
 Used on: Service
 
-This annotation is used for enabling [Topology Aware Hints](/docs/concepts/services-networking/topology-aware-hints/) for a service. This tells the EndpointSlice controller to set topology hints if it is deemed safe.
+This annotation is used for enabling [Topology Aware Hints](/docs/concepts/services-networking/topology-aware-hints/) for a Service.
+If you set the annotation to `"auto"`, that instructs the EndpointSlice controller to set topology hints
+(if the control plane considers that appropriate).
+
+There are no other valid values for this annotation. If you don't want topology aware hints
+for a Service, don't add this annotation.
```

Comment:
I recommend tech review on this revised wording - @sergeyshevch if you're confident that this behavior is as written (**_I_** suggested it), then please reply to confirm.

Reply:
I have no huge confidence about this rewrite. We are in the progress of adopting Topology Aware Hints in our cluster and have not properly tested them.

But your rewrite totally matches with topology-aware hits docs, so it should be fine.

Reply:
@sftim What do you think here? We may need to request someone's other review.

Reply:
This is ready for a tech review by SIG Network.